pkg/apis/serving/v1beta1/predictor_tfserving.go

```diff
@@ -28,13 +28,12 @@ import (
 )
 
 var (
-	TensorflowEntrypointCommand          = "/usr/bin/tensorflow_model_server"
-	TensorflowServingGRPCPort            = "9000"
-	TensorflowServingRestPort            = "8080"
-	TensorflowServingGPUSuffix           = "-gpu"
-	InvalidTensorflowRuntimeVersionError = "Tensorflow RuntimeVersion must be one of %s"
```

Comment:
`InvalidTensorflowRuntimeVersionError` is not used anymore right?

Reply:
yes